store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java

```diff
@@ -1278,7 +1282,8 @@ public void doDispatch(DispatchRequest req) {
         }
     }
 
-    public void putMessagePositionInfo(String topic, int queueId, long offset, int size, long tagsCode, long storeTimestamp,
+    public void putMessagePositionInfo(String topic, int queueId, long offset, int size, long tagsCode,
+        long storeTimestamp,
```

Comment:
@vongosling , I am not sure what in checkstyle trigger this. And what is **UT**? Sorry, no clue. 😝